### PR TITLE
feat: setup balrog cluster with thoth namespace, kafka and prometheus

### PR DESCRIPTION
**Pull request by harshad16**
setup balrog cluster with thoth namespace, kafka and prometheus
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

File: cluster-scope/overlays/prod/emea/balrog/kustomization.yaml [hunk 1]
```diff
@@ -15,3 +30,4 @@ generators:
 patchesStrategicMerge:
     - oauths/cluster_patch.yaml
     - groups/cluster-admins.yaml
+    - subscriptions/prometheus-operator_patch.yaml
```
Comment (HumairAK) [inline cluster-scope/overlays/prod/emea/balrog/kustomization.yaml:33]: why not use user workload monitoring?

Comment (harshad16) [reply cluster-scope/overlays/prod/emea/balrog/kustomization.yaml:33]: what is that @HumairAK ?

Comment (harshad16) [reply cluster-scope/overlays/prod/emea/balrog/kustomization.yaml]: thanks i have removed it

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

**Pull request merged**